packages/util/src/crypt.ts

```diff
@@ -154,34 +154,34 @@ export const base64 = {
    *
    * @param input An array of bytes (numbers with
    *     value in [0, 255]) to encode.
-   * @param opt_webSafe Boolean indicating we should use the
+   * @param optwebSafe Boolean indicating we should use the
```

Comment:
Did you mean to get rid of the `opt_` entirely? https://engdoc.corp.google.com/eng/doc/devguide/js/styleguide.md?cl=head#features-functions-default-parameters

If not, it should still be camelCase I think.

Reply:
Good catch. Updated to `webSafe`.

Reply:
I think they mean `option` instead of `optional` which the google styleguide references to, but it is redundant.